book/layout.md

```diff
@@ -142,6 +142,34 @@ tree][gecko-tree].
 [webkit-tree]: https://webkit.org/blog/114/webcore-rendering-i-the-basics/
 [gecko-tree]: https://wiki.mozilla.org/Gecko:Key_Gecko_Structures_And_Invariants
 
+::: {.mc-quiz #layout-tree-quiz}
```

Comment:
```suggestion
::: {.web-only .mc-quiz #layout-tree-quiz}
```

Reply:
Also: Please put these imports behind a default-off flag configured by config.json.

Reply:
I'll put the imports behind a default-off flag. How about I make the `.mc-quiz` class act like `.web-only` by default, since the quizzes won't make sense anywhere else?

Reply:
I've added a `show_quiz` flag; it's on only in the `draft` mode. Assets and quizzes only get emitted when in `draft` mode.